.github/workflows/nms-workflow.yml

```diff
@@ -0,0 +1,255 @@
+    steps:
+      - uses: actions/checkout@v2
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: install  # will run `yarn install` command
+      - name: run yarn test
+        run: |
+          cd ${MAGMA_ROOT}/nms
+          yarn add jest@^26.4.2 -W --dev
+          yarn test:ci
+      # Notify ci channel when failing
+      # Plugin info: https://github.com/marketplace/actions/slack-notify
+      - name: Notify failure to slack
+        if: failure() && github.event_name == 'push'
+        uses: rtCamp/action-slack-notify@v2.2.0
+        env:
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_CI }}
+          SLACK_TITLE: "NMS yarn test failed"
+          SLACK_MESSAGE: "${{ steps.commit.outputs.title}}"
+          SLACK_USERNAME: "NMS workflow"
+          SLACK_ICON_EMOJI: ":boom:"
+          SLACK_COLOR: "#FF0000"
+          SLACK_FOOTER: ' '
+  nms-e2e-test:
+    needs:
+      - path_filter
+      - nms-flow-test
+      - nms-eslint
+      - nms-yarn-test
+    if: ${{ needs.path_filter.outputs.should_skip == 'false' }}
+    name: nms-e2e-test job
+    runs-on: ubuntu-latest
+    env:
+      MAGMA_ROOT: "${{ github.workspace }}"
+      NMS_ROOT: "${{ github.workspace }}/nms/packages/magmalte"
+      PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: "true"
+      PUPPETEER_EXECUTABLE_PATH: "/usr/bin/google-chrome-stable"
+    steps:
+      - uses: actions/checkout@v2
+      - name: apt install yarn
+        run: |
+            cd ${MAGMA_ROOT}/nms
+            sudo apt-get install -y apt-transport-https
+            curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+            echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+            sudo apt-get update -y
+            sudo apt-get install -y yarn
+            yarn add jest@^26.4.2 -W --dev
+      - name: apt install chrome
+        run: |
+            cd ${MAGMA_ROOT}/nms
+            # Install latest chrome dev package
+            # Note: this installs the necessary libs to make the bundled version of Chromium that Puppeteer
+            # installs, work.
+            curl -sS https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+            echo "deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main" | sudo tee /etc/apt/sources.list.d/google.list
+            sudo apt-get update -y
+            sudo apt-get install -y google-chrome-stable libxss1 --no-install-recommends
+            sudo rm -rf /var/lib/apt/lists/*
+      - uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: install  # will run `yarn install` command
+      - name: run e2e_test_setup.sh
+        run: |
+            source $NVM_DIR/nvm.sh
+            nvm install stable
+            cd ${MAGMA_ROOT}/nms/packages/magmalte
+            ./e2e_test_setup.sh
+      - name: Publish Unit Test Results
+        uses: EnricoMi/publish-unit-test-result-action@v1
```

Comment:
we should use the unit-test-workflow to publish the test results as the pull request event will not have write permission.
for example see here https://github.com/magma/magma/blob/9c186ea252172ed32036f4e08e52c1772d0e8926/.github/workflows/cloud-workflow.yml#L161 and here https://github.com/magma/magma/blob/master/.github/workflows/unit-test-workflow.yml

Reply:
thank you!.. fixed

Reply:
you did not added the nms workflow to the trigger list in there https://github.com/magma/magma/blob/master/.github/workflows/unit-test-workflow.yml

Reply:
thanks for catching this!.. i missed it